.maintain/runtime-weight-template.hbs

```diff
@@ -16,23 +16,12 @@
 use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
 use sp_std::marker::PhantomData;
 
-/// Weight functions needed for {{pallet}}.
```

Comment:
We needed to be using `WeightInfo` from the pallet, not a local version. This was preventing us from actually using the weights we generated for our runtime, for external pallets.